packages/blade/src/components/ProgressBar/progressBarTokens.ts

```diff
@@ -0,0 +1,18 @@
+const indeterminateAnimation = {
+  scaleXInitial: 1,
+  scaleXMid: 5,
+  scaleXFinal: 1,
+  leftInitial: '-8%',
+  leftMid: '50%',
+  leftFinal: '103%',
+  fillWidth: '5%',
```

Comment:
Are these values arbitary or comes from the motion design?

Reply:
They're from motion design. We need it to go precisely -8% to precisely 103%